src/components/screen/Oauth/Oauth.vue

```diff
@@ -7,9 +7,12 @@
     <div v-if="isInited">
       <section class="oauth-section">
         <v-card class="card-content">
+          <logout-button class="oauth-logout" />
           <form-field>
-            <server-switcher @switch="onServerSwitch" />
-            <logout-button class="oauth-logout" />
+            <server-switcher
+              v-if="isServerSwitcherVisible"
```

Comment:
@ifiri move this `v-if="isServerSwitcherVisible"` to parent `<form-field>`.
we don't need soo much empty space
![image](https://user-images.githubusercontent.com/756731/85378607-721a8b80-b543-11ea-8f49-3830ae69290c.png)

it should be like this
![image](https://user-images.githubusercontent.com/756731/85378677-8b233c80-b543-11ea-87b9-2f8bb1b6d4d9.png)

Reply:
done